src/pages/Polls.jsx

```diff
@@ -11,18 +12,31 @@ const { Title } = Typography;
 const Polls = (props) => {
   const [polls, setPolls] = useState([]);
   const [isPollsLoading, setIsPollsLoading] = useState(false);
+  const user = useSelector((state) => state.user);
 
   useEffect(() => {
     getPolls();
   }, []);
 
   async function getPolls() {
     setIsPollsLoading(true);
-    const response = await PollService.getUserPolls(props.statuses);
+    const response = user.role === 'RESPONDENT' 
+      ? await PollService.getRespondentPolls(props.statuses) 
+      : await PollService.getManagerPolls();
     setPolls(response);
     setIsPollsLoading(false);
   }
 
+  function getLink(pollStatus){
+    if(pollStatus === "OPEN" || pollStatus === "PROGRESS"){
+      return `/availablePolls/`
+    } else if (user.role === 'MANAGER'){
+      return `/overallRating/`
+    } else if (user.role === 'RESPONDENT'){
+      return `/profile/`
+    }
+  }
```

Comment:
пусть функция всегда что-то возвращает
```suggestion
  function getLink(pollStatus){
    if (pollStatus === "OPEN" || pollStatus === "PROGRESS"){
      return `/availablePolls/`
    } else if (user.role === 'MANAGER'){
      return `/overallRating/`
    } 
    return `/profile/`
  }
```